packages/samples/src/demos/tutorialAPICalls.ts

```diff
@@ -199,6 +199,50 @@ cl.AddCallback({
     }
 })
 
+cl.AddCallback({
+    name: 'MyCallbackWithStubs',
```

Comment:
Might have to remove before merging until we decide plan for tutorial. Would need to add entities to support it.